.golangci.example.yml

```
@@ -878,8 +882,8 @@ linters-settings:
         severity: warning
         disable: false
         arguments:
-          - - ID
-          - - VM
+        - [ "ID" ] # Whitelist
+        - [ "VM" ] # Blacklist
```

my syntax works :wink: , but if you prefer `- ["xx"]` is also possible.
```suggestion
        - [ "ID" ] # Allowlist
        - [ "VM" ] # Denylist
```

---

I didn't said it not works, but whitelist and blacklists was discovery for me (and I spend some time checking what is what in revive). So this is important thing to mention, imo.